GitVersionExe.Tests/ArgumentParserTests.cs

```diff
@@ -211,4 +211,11 @@ public void can_log_to_console()
         var arguments = ArgumentParser.ParseArguments("-l console -proj foo.sln");
         arguments.LogFilePath.ShouldBe("console");
     }
+
+    [Test]
+    public void nofetch_true_when_defined()
```

Comment:
Can you make sure arguments after this get parsed properly. The parser is not that great and switches have created issues in the past